### PR TITLE
Clean up logging on manual review.

### PR DESCRIPTION
**Pull request by shaper**
Optional follow-up tracked in #5551

### CODE REVIEW

File: java/arcs/core/storage/RawEntityDereferencer.kt [hunk 1]
```diff
@@ -36,7 +36,8 @@ class RawEntityDereferencer(
     private val entityActivationFactory: ActivationFactory? = null,
     private val referenceCheckFun: ((Schema, RawEntity?) -> Unit)? = null
 ) : Dereferencer<RawEntity> {
-    private val log = TaggedLog { "Dereferencer(${schema.names})" }
+    // TODO(#5551): Consider including a hash of schema.names for easier tracking.
+    private val log = TaggedLog { "RawEntityDereferencer)" }
```
Comment (jasonwyatt) [inline java/arcs/core/storage/RawEntityDereferencer.kt:40]:
```suggestion
    private val log = TaggedLog { "RawEntityDereferencer" }
```

Comment (shaper) [reply java/arcs/core/storage/RawEntityDereferencer.kt]: Done

**Pull request merged**